src/pages/Profile/components/UserPosts/UserPostFilter/UserPostFilter.Styles.ts

```diff
@@ -0,0 +1,34 @@
+import styled from "styled-components"
+
+export const UserPostFilterLayout = styled.div<{ $isMyPage: boolean }>`
+  position: absolute;
+  left: 50%;
+  transform: ${({ $isMyPage }) =>
+    $isMyPage ? `translateX(-47%)` : `translateX(-50%)`};
+
+  display: flex;
+  gap: 2rem;
+`
+
+export const UserPostButton = styled.button<{
+  $isSelected?: boolean
+}>`
+  padding: 0.8rem 1.2rem;
+
+  font-size: ${({ theme }) => theme.fontSizes.semiLarge};
+
+  border-radius: ${({ theme }) => theme.radius.size8};
+
+  user-select: none;
+
+  background-color: ${({ theme, $isSelected }) =>
+    $isSelected ? theme.colors.point_alt : theme.colors.point};
+
+  &:hover {
+    background-color: ${({ theme }) => theme.colors.point_alt};
```

Comment:
저도 재영님처럼 hover시에는 transition 효과를 넣으면 색상이 자연스럽게 변해서 보기 좋은 것 같더라고요! 🤔 
만약 바꾸신다면 이렇게 바꿔보시면 될 것 같슴다! 😊
```suggestion
    background-color: ${({ theme }) => theme.colors.point_alt};
    transition: 0.3s ease-in-out;
```